### ISSUE TITLE
The tile disappears in 2.19

### ISSUE THREAD

**Issue by GeorgeYves** [mueller-ma/Coffee#262]
My device is smartphone Samsung galaxy A8+, Android 9. I installed the app from F-Droid. The latest version was 2.18 when I updated to 2.19. After the update I saw that the tile was empty and didn't work. I restarted my phone but the tile disappeared completely. I tried to do a fresh installation of 2.19 but the tile didn't appear. Now I have to re-install version 2.18.

**Comment by mueller-ma** [comment]
That's really strange. For some users the tile re-appeared after updating to 2.19.

I have no idea what causes this issue.

**Comment by DimiPaparas** [comment]
I am also experiencing this issue. I believe I installed version 2.20, then uninstalled it and reinstalled only to find that the tile is functional but lacks an icon.

I'm suspecting it has to do with where the icon is stored. I'll look into it a bit more.

I've attached an image of the dialog now in version 2.21 - which is not available from F-Droid.

![Screenshot_2023-11-03-10-41-38-01_959e28baa4b356007146fb0e222e2f5d.jpg](https://github.com/mueller-ma/Coffee/assets/87274658/f26184ae-9365-4387-bc6d-e65547cfe0c6)
